src/components/dashboard/FaceRecognition/FaceRecognition.js

```diff
@@ -41,7 +37,7 @@ type State = {
 class FaceRecognition extends React.Component<FaceRecognitionProps, State> {
   state = {
     showPreText: false,
-    showZoomCapture: true,
+    showCamera: true,
```

Comment:
Also please, review this component code and remove all method/instance variables aren't needed anymore (for example, timerId)

Reply:
done in https://github.com/GoodDollar/GoodDAPP/pull/1341/commits/44a0fdf04a8750dde4707827f97a7d4732e4e649